docs/comment_copy_guidelines.md

```diff
@@ -0,0 +1,133 @@
+analyzer (those that can become output), are comments on the `master` branch, on 
+the  [`exercism/website-copy`][git-website-copy] repository.
+
+At the moment of writing, [this issue][issue-ci-comments] tracks the status of any
+generalisation of this CI, if any.
+
+## Wording
+
+- Avoid unnecessarily wordy comments. Be concise.
+- Be neutrally observational; avoiding charged statements and blanket statements.
+- Make the recommendation explicit.
+- When possible, put the recommendation first, then the explanation.
+- Avoid "me", "I", "we", etc, since the bot is not a person.
+- Avoid "you" and "your code", because it can sometimes come across as judging 
+  the person rather than the code.
+- Avoid words like "just", "simply", "obviously", which can come across as 
+  condescending: if the comment is necessary, it was _obviously_ not _obvious_.
+- Avoid making assumptions about what people know and don't know. The only
+  exception is knowledge from already completed _core_ exercises. Avoid
+  "as you know", "as you remember", "as you learned", "now that we all
+  understand x", because even though something was said, the person doesn't 
+  necessarily understand it.
+
+### Guidance
+
+For the first few exercises of a track, the following is _extra_ important:
+
+- **Name the difference** between what the person is doing, and what is
+  'idiomatic' in the language.
+- **Use the proper terms** and nomenclature, so people can recognize those
+  concepts elsewhere, and also can research them by themselves.
+- **Keep it relatively short**, avoiding a wall of text or overwhelming them
+  with advice. If they have a great experience in the first exercise, they'll
+  come back and you'll have many more opportunities to provide feedback on all
+  of the things that you noticed.
+- **Don't give the solution** as general advice for all Exercism mentoring. 
+  Learning sticks when people discover the answer for themselves. That's a 
+  tremendously exhilarating experience, and the emotional kick makes it 
+  memorable. However, if the discovery **doesn't** trigger a dopamine hit, it 
+  makes total sense to show what it looks like.
+- **Don't overly explain a concept**: don't go in depth about the underlying 
+  mechanisms of compilers and things. In this case it's more about this being 
+  one of the first exercises in the language track, and at this stage feedback
+  is more helpful if it's shorter and more directive.
+- **Give a link** that shows exactly how to do the concept, in a **tutorial** 
+  kind of way. Meaning: showing how to do things, not discussing why. This might
+  mean that the official docs of the language don't suffice, as these are often
+  a code reference and don't show you how to use it and how it works. _However_,
+  you should provide a link only for more in-depth exploration. The person
+  should be able to understand what you mean from the answer directly, without
+  following the link.
+
```

Comment:
**Congratulate the student** They presumably worked hard in completing the exercise and learned something; they deserve some encouragement. Find something positive to say about their code if possible.

Reply:
I am not sure about this, as the comments are used on `auto-mentoring`. It would be a bot saying: "You have done great!".

But I am very interested what @kytrinyx thinks about this.

Reply:
I agree with @tehsphinx here. I feel that the tone of an automated mentor should be different from a personal mentor, to help make the distinction between what is automated mentoring and what is personal mentoring. As such, I don't feel that the automated mentor should be made more human by adding very human-like sentences. Human mentors should absolutely do such things, but I'm not sure the automated mentor should.

Reply:
If we do choose to praise people (I'm in favour of this, but I seem to be in the minority) then I think this would happen as a covering note of the comments, rather than in the comments themselves. For example, at the moment we say words to the effect of "Your solution has been approved by our automated analyzer. Congratulations! Here are some comments that might also help you."

Reply:
> "Your solution has been approved by our automated analyzer. Congratulations! Here are some comments that might also help you."

If you verse it like this, I'm perfectly fine with that. It's just that I think we have to be really careful to don't give the impression that there is a human being on the other side.

Reply:
I think as part of the "be neutral" guideline, we don't want to add emotion to the _comments_. I think as proposed above, I really like the idea of adding it to the "auto-approve summary message".

Reply:
> Find something positive to say about their code if possible.

For what it's worth, I generally am _very_ positive when mentoring, because every submission is a step forward and I think encouragement is great, but saying something positive for the sake of something positive, in a similar manor as to how _some_ people (and companies) want PR reviews to go, is not something that is appreciated _in every culture_.

For example: In a large cross-section of my personal circles, empty positivity (like "The Praise Sandwich") for the sake of saying something positive, is considered **extremely rude** because it's ingenuous and completely removes meaning which detracts from anyone _actually being positive_.

Begin neutral is generally safer.